v3.0/tidb-in-kubernetes/deploy/gcp-gke.md

```diff
@@ -55,16 +55,15 @@ gcloud services enable container.googleapis.com
 
```

Comment:
^ This gcloud command takes multiple arguments, so this can be compressed into one line (maybe use 2 or 3 for better readability?)